Source/MNASolver.cpp

```diff
@@ -80,7 +80,7 @@ void MnaSolver<VarType>::initialize() {
 	// Some components feature a different behaviour for simulation and initialization
 	for (auto comp : mSystem.mComponents) {
 		auto powerComp = std::dynamic_pointer_cast<CPS::TopologicalPowerComp>(comp);
-		if (powerComp) powerComp->setBehaviour(TopologicalPowerComp::Behaviour::Simulation);
+		if (powerComp) powerComp->setBehaviour(TopologicalPowerComp::Behaviour::MNASimulation);
```

Comment:
@dinkelbachjan Why do we need to differentiate between Simulation and MNASimulation?

Reply:
The `Simulation` behaviour has been removed, while `MNASimulation` and `PFSimulation` have been introduced instead, see

https://github.com/sogno-platform/dpsim/blob/5e36b121b1906f432e782eec807aefbc8e5fee5b/models/Include/cps/TopologicalPowerComp.h#L26

This has been done to be able to be more specific on the type of simulation. In particular, component models from the SP domain are used at the same time for MNA and *standalone* PF simulations. The component model can look different, e.g. snubber components should only be added for MNA simulations, but not for *standalone* PF simulations. An example for that you can find here 

https://github.com/sogno-platform/dpsim/blob/5e36b121b1906f432e782eec807aefbc8e5fee5b/models/Source/SP/SP_Ph1_Transformer.cpp#L129